nulib/dc-nextjs
# Add support for private or institutional works display on Work page

## What does this do
On the Work page, handles scenarios for Work's which require authentication
- Adds a blurred image component, using a Work's `thumbnail` as the source.
- Adds an Announcement message indicating why there is a blurred image.
- Adds `visibility` facet, so we can easily find "Institution" and "Private" Works
- Updates the Northwestern favicon in browser

![image](https://user-images.githubusercontent.com/3020266/198363454-5e297279-6e34-47b8-aa8b-8b7745097f45.png)

## How to test
1. Go to search page
2. Filter by "Rights and Usage" > "Visibility"
3. Select an "Institution" Work
4. Verify blurred image (only will appear if a `thumbnail` value is present, which we think will eventually get populated for all Works?)
5. Click "sign in" link in the announcement, and you'll scroll to top of the viewport

## Other notes
the `<BlurredBgImage />` component is somewhat flexible and we may be able to use this in other places.  Here's the component's Props types:

```
interface Props {
  bgColor?: string;
  bgImageUrl: string;
  height?: string;
  width?: string;
}
```
